content/fis-i/exercicios/0999-fichas-2021-2022.md

```diff
@@ -42,7 +42,7 @@ Por razões óbvias, as resoluções apenas serão disponibilizadas após o fech
 
 - Semana 6: Termodinâmica (temperatura, dilatação, estados da matéria)
 
-  - Resolução Diogo Correia ([Quizz 13](https://drive.google.com/file/d/1xhw5nNW_jZwFVVeKg_2U4jmY5SSsGNBp/view) | [TP11](https://drive.google.com/file/d/1ToD6NGEUHEldeClOHkqUyep9WoSK6y4U/view))
+  - Resolução Diogo Correia ([Quiz 13](https://drive.google.com/file/d/1xhw5nNW_jZwFVVeKg_2U4jmY5SSsGNBp/view) | [TP11](https://drive.google.com/file/d/1ToD6NGEUHEldeClOHkqUyep9WoSK6y4U/view))
```

Comment:
I know it's incorrect, but it's what the teachers called them. We kept it for consistency
```suggestion
  - Resolução Diogo Correia ([Quizz 13](https://drive.google.com/file/d/1xhw5nNW_jZwFVVeKg_2U4jmY5SSsGNBp/view) | [TP11](https://drive.google.com/file/d/1ToD6NGEUHEldeClOHkqUyep9WoSK6y4U/view))
```